vuepress/docs/next/docs/concepts/operator-intro.md

```diff
@@ -0,0 +1,93 @@
+# The Entando Operator
+
+The Entando Operator processes the custom resources in Kubernetes that represent the different [components of an Entando application](./README.md).
+The goal of the operator is to provide automation and a set of default configuration options to simplify and accelerate the deployment of an Entando application.
+
+The sections below provide details and assumptions that the operator makes when deploying Entando Custom Resources. If you're using OpenShift these
+sections will provide details on how to configure your deployment via the Operator Hub.
+
+For details on the individual custom resources and their configuration check out the [custom resources documentation](./custom-resources.md).
+
+## Installation Scope Options
+
+The Entando Operator can be installed in one of two modes. Either cluster scoped or namespace scoped. In a cluster scoped deployment the operator will have
+visibility across the cluster and will manage all of the Entando applications in the Kubernetes cluster. In a namespace scoped deployment the operator
+will only have visibility to the namespace where it is deployed and will manage only the components in that namespace.  
+
+When installing the operator via OpenShift look for the `Installation Mode` option to select the scoping for the operator.
+
+### When to use Cluster Scoped Deployments
+When choosing how to deploy your operator there are no right or wrong answers. Think about the deployment that best fits your goals and team. Here are some items to think about.
+
+- Cluster scoped deployments are common in production clusters and in environments with strong operational support and controls.
+- A cluster scoped deployment can saves resources When you want to optimize resource consumption and share Kubernetes infrastructure.
+- When you are planning to centralize and share other infrastructure resources like Keycloak and databases a cluster scoped deployment can fit into the same management processes.
+   - Sharing infrastructure resources is a recommended approach for medium or large teams and the operator can be treated like other infrastructure services.
+- In some cases security requirements will require that the permissions required for the operator are managed separately from the deployed applications. A cluster scoped deployment isolates the operator permissions in a separate namespace.
+- Cluster scoped deployments can simplify the deployment of an Entando app by developers or end users because they have fewer resources to manage
+
+###  When to use Namespace Scoped Deployments
+When choosing how to deploy your operator there are no right or wrong answers. Think about the deployment that best fits your goals and team. Here are some items to think about.
```

Comment:
Minor - common sentences here. Maybe move it above as an intro to the sub-sections.

Reply:
Updated